Specification/3._Data_structure_description_language.md

```diff
@@ -132,7 +115,7 @@ Each component is discussed below.
 
 #### Field type
 
-Can be either a primitive data type (primitive data types are defined below) or a nested data structure.
+Fields can be either a primitive data type (primitive data types are defined below) or a nested data structure.
```

Comment:
"Type" is a property of the entity "Field". Saying that fields can be a data type is incorrect. I propose the following wording:

> Fields can be either of a primitive data type

Note the "of". Although I'm not against any equivalent wording here.